web/src/reload.ts

```diff
@@ -21,7 +20,16 @@ import * as util from "./util.ts";
 
 let server_reachable_check_failures = 0;
 
-const token_metadata_schema = z.object({url: z.string(), timestamp: z.number()});
+export const reload_metadata_schema = z.object({
+    reload_data: z.object({
```

Comment:
Hmm, why is there a `reload_data` cousin to `timestamp`? I'm not sure I see the purpose of that extra nesting.